aws/README.md

```diff
@@ -6,16 +6,31 @@
 3. [AWS IAM Authenticator](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html)
 
 ### Set Your Variables
-1. Clone this repository to your local machine.
-2. Uncomment and replace variables in `terraform.tfvars` with the desired values. Check `variables.tf` for default values.
+1. Fork and clone this repository to your local machine.
+2. `cd` into `./aws`, then uncomment and replace variables in `terraform.tfvars` with the desired values. Check `variables.tf` for default values.
```

Comment:
Safe to assume we're already in `aws` at this point?

Reply:
Yup you're right.